src/windowsservercore/ltsc2022/helix/webassembly-net8/amd64/Dockerfile

```diff
@@ -1,11 +1,13 @@
+# escape=`
+
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-ltsc2022-helix-amd64
 
 # Install 7zip
-ENV ZIP7_VERSION=1900
+ENV ZIP7_VERSION=2408
```

Comment:
In theory, we could use the GitHub releases API to determine the latest version from https://github.com/ip7z/7zip/releases. I'm unsure of the "officialness" of that repo. But it could just be used to get the version and then still install from 7-zip.org.

Reply:
I am going to log a more follow-up issue on this.